hypertrace-gradle-docker-plugin/src/main/java/org/hypertrace/gradle/docker/DockerPlugin.java

```diff
@@ -109,6 +109,7 @@ private String getBranchTag() {
     // to build in CI), otherwise for local builds use 'test'
     return getEnvironmentVariable("CIRCLE_BRANCH")
         .map(String::trim)
+        .map(branch -> branch.replaceAll("[^A-Za-z0-9._-]", ""))
```

Comment:
Don't you need to escape `.` and `-`?

Reply:
Tested. It works without escaping as well https://regex101.com/r/t4E98P/1

Reply:
still escaped them to be safe